src/org/omegat/Main.java

```diff
@@ -264,14 +264,19 @@ public static void restartGUI(String projectDir) {
             command.addAll(CLIParameters.unparseArgs(PARAMS));
         } else {
             // assumes jpackage
-            javaBin = Paths.get(StaticUtils.installDir()).getParent().resolve("bin/OmegaT");
-            if (!javaBin.toFile().exists()) {
-                // abort restart
-                Core.getMainWindow().displayWarningRB("LOG_RESTART_FAILED_NOT_FOUND");
+            var installDir = StaticUtils.installDir();
+            if (installDir == null) {
                 return;
+            } else {
+                javaBin = Paths.get(installDir).getParent().resolve("bin/OmegaT");
+                if (javaBin.toFile().exists()) {
```

Comment:
The logic become reverted. The change led the restart on jpackage distribution become failure.